MonoGame.Framework/DrawableGameComponent.cs

```diff
@@ -70,8 +66,14 @@ public override void Initialize()
 
         public virtual void Draw(GameTime gameTime) { }
 
-        protected virtual void OnVisibleChanged(object sender, EventArgs args) { }
+        protected virtual void OnVisibleChanged(object sender, EventArgs args)
+        {
+            EventHelpers.Raise(this, VisibleChanged, args);
+        }
 
-        protected virtual void OnDrawOrderChanged(object sender, EventArgs args) { }
+        protected virtual void OnDrawOrderChanged(object sender, EventArgs args)
+        {
+            EventHelpers.Raise(this, DrawOrderChanged, args);
+        }
```

Comment:
This and the change to `OnVisibleChanged` is a good bug fix too.  As the docs state:

> Called when the DrawOrder property changes. Raises the DrawOrderChanged event. 
https://msdn.microsoft.com/en-us/library/microsoft.xna.framework.drawablegamecomponent.ondraworderchanged.aspx

> Called when the Visible property changes. Raises the VisibleChanged event. 
https://msdn.microsoft.com/en-us/library/microsoft.xna.framework.drawablegamecomponent.onvisiblechanged.aspx

We were incorrectly not calling the event from the virtual before.